heaps/min_heap.py

```diff
@@ -46,10 +78,17 @@ def __str__(self):
 
     def empty(self):
         """ This method returns true if the heap is empty
-            Time complexity: ?
-            Space complexity: ?
+
+            Time complexity: O(1) constant because the method only needs to calculate the length of the array
+            each time to perform its function and the en() function in Python runs in O(1) complexity.
+
+            Space complexity: O(1) beacause only one new variable (heap_size) is created each time the function is called. 
         """
-        pass
+        heap_size = len(self.store)
+
+        if heap_size == 0:
+            return True
+        return False
 
 
     def heap_up(self, index):
```

Comment:
👍  This function works and you have the time/space complexity right, but it is a little overcomplicated.  

Instead of that for loop you could do something like

`while current_index > 0 and self.store[current_index] < self.store[parent_index]:`